WcaOnRails/config/locales/locales.rb

```diff
@@ -1,4 +1,4 @@
-# frozen_string_literal: true
+﻿# frozen_string_literal: true
```

Comment:
This is fascinating. The two lines look identical, don't they? Check out the hex dump:

```
~/gitting/worldcubeassociation.org/WcaOnRails @kaladin> xxd config/locales/locales.rb | head
00000000: efbb bf23 2066 726f 7a65 6e5f 7374 7269  ...# frozen_stri
00000010: 6e67 5f6c 6974 6572 616c 3a20 7472 7565  ng_literal: true
00000020: 0a6d 6f64 756c 6520 4c6f 6361 6c65 730a  .module Locales.
00000030: 2020 4156 4149 4c41 424c 4520 3d20 7b0a    AVAILABLE = {.
00000040: 2020 2020 2265 6e22 3a20 7b0a 2020 2020      "en": {.    
00000050: 2020 2266 6c61 675f 6964 223a 2022 6762    "flag_id": "gb
00000060: 222c 0a20 2020 2020 2022 6e61 6d65 223a  ",.      "name":
00000070: 2022 456e 676c 6973 6822 2c0a 2020 2020   "English",.    
00000080: 7d2c 0a20 2020 2022 6573 223a 207b 0a20  },.    "es": {. 
00000090: 2020 2020 2022 666c 6167 5f69 6422 3a20       "flag_id": 
```

Those first three bytes (0xEF, 0xBB, 0xBF) are a BOM! See http://stackoverflow.com/a/2223926.

I'm going to look into updating our precommit hook to not allow BOM markers (#1218).